Improved markdown along with improved description!

devmike123 opened the pull request:
I've improved markdown and added more description in first of the functions and will be doing so in the future.

Hope your effort reduces with my contribution!

devmike123 committed:
Fixed author.md

Made author.md more descriptive with language and markdown improvement!

devmike123 committed:
Merge pull request #1 from DevMike123/fixed-author

Fixed author.md

devmike123 committed:
Update usdauthor.md

devmike123 committed:
Update usdaddfields.md

Fixed addFields description!

devmike123 committed:
Update usdauthor.md

devmike123 committed:
Update usdaddfields.md

devmike123 committed:
Update usdaddemoji.md

devmike123 committed:
Update usdaddemoji.md

devmike123 committed:
Update usdaddemoji.md

devmike123 committed:
Update usdaddreactions.md

devmike123 committed:
Update usdallemojicount.md

devmike123 committed:
Update usdabbreviate.md

devmike123 committed:
Update usdabbreviate.md

devmike123 committed:
Update usdabbreviate.md

devmike123 committed:
Update usdactivity.md

devmike123 committed:
Update usdaddbutton.md

devmike123 committed:
Update usdaddcmdreactions.md

devmike123 committed:
Update usdaddfield.md

devmike123 committed:
Update usdaddfield.md

devmike123 committed:
Update usdaddfields.md

devmike123 committed:
Update usdaddmessagereactions.md

devmike123 committed:
Update usdaddobjectproperty.md

devmike123 committed:
Update usdaddtimestamp.md

devmike123 committed:
Update usdadvancedtextsplit.md

devmike123 committed:
Update usdallchannelscount.md

devmike123 committed:
Update usdallmemberscount.md

devmike123 committed:
Update usdargscheck.md

devmike123 committed:
Update usdattachment.md

devmike123 committed:
Update usdawaitcmdreaction.md

devmike123 committed:
Update usdawaitmessages.md

devmike123 committed:
Update usdawaitreaction.md

devmike123 committed:
Update usdawaitreactionsuntil.md

devmike123 committed:
Update usdban.md

devmike123 committed:
Update usdbancount.md

devmike123 committed:
Update usdblacklistids.md

devmike123 committed:
Update usdblacklistroleids.md

devmike123 committed:
Update usdblacklistserverids.md

devmike123 committed:
Update usdboostingsince.md

devmike123 committed:
Update usdbotcount.md

devmike123 committed:
Update usdbotlastmessagechannelid.md

devmike123 committed:
Update usdbotlastmessageid.md

devmike123 committed:
Update usdbotleave.md

devmike123 committed:
Update usdbotownerid.md

devmike123 committed:
Update usdbottyping.md

devmike123 committed:
Update usdcachemembers.md

devmike123 committed:
Update usdcategorychannels.md

devmike123 committed:
Update usdchangenickname.md

devmike123 committed:
Update usdchannel.md

devmike123 committed:
Update usdchannelcategoryid.md

devmike123 committed:
Update usdchannelcooldown.md

devmike123 committed:
Update usdchannelcount.md

devmike123 committed:
Update usdchannelexists.md

devmike123 committed:
Update usdchannelid.md

devmike123 committed:
Update usdchannelmessageexists.md

devmike123 committed:
Update usdyear.md

devmike123 committed:
Update usdyear.md

devmike123 committed:
Update usdwebhookexists.md

devmike123 committed:
Update usdwebhook.md

devmike123 committed:
Update usdchannelname.md

devmike123 committed:
Update usdchannelnsfw.md

devmike123 committed:
Update usdchanneloverwrites.md

devmike123 committed:
Update usdchannelpermissionsfor.md

devmike123 committed:
Update invite-system.md

devmike123 committed:
Update channel-variables.md

devmike123 committed:
Update global-variables.md

devmike123 committed:
Update message-variables.md

devmike123 committed:
Update server-variables.md

devmike123 committed:
Update user-variables.md

devmike123 committed:
Add files via upload

devmike123 committed:
Delete aoi_banner.png.jpg

devmike123 committed:
Add files via upload

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update home.md

Leref commented on src/home.md:
```diff
@@ -1,16 +1,18 @@
 ---
-coverY: 0
```
This is due to the cover page and Gitbook itself.

Leref commented on README.md:
```diff
@@ -1,12 +1,12 @@
+**Welcome To The Home Page Of aoi.js Documentation.**
 
 ## Introduction
+<img src = "./.gitbook/assets/aoi_banner.jpg">
```
This should be reversed as the next version comes, the website of aoi.js will have updated banner unlike here will be outdated since the URL.

devmike123 replied:
What's the updated banner of aoi.js?

devmike123 committed:
Add files via upload

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update usdchannelsendmessage.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update usdawaitcmdreaction.md

devmike123 committed:
Added data? to the file.

devmike123 committed:
Update usdactivity.md

devmike123 committed:
Update to v5.

devmike123 committed:
Update usdhasperms.md

devmike123 committed:
Updated to v5!

Leref commented on README.md:
````diff
@@ -1,63 +1,139 @@
+```js
+bot.command({
+name: "play", //Trigger name (command name)
+code: `$playSong[song;leave vc time;defean (yes or no);leave when vc empty (yes/no);error]`
````
Outdated.

Leref commented on README.md:
```diff
@@ -1,63 +1,139 @@
+
+## Links
+
+Aoi.JS was made by [Aoi.JS Team](https://aoi.js.org)
```
Outdated.

Leref commented on README.md:
```diff
@@ -1,63 +1,139 @@
+        channel: "Channel ID", //Enter a Channel ID
+        code: `<@$authorID> just joined, welcome!` //This can be changed
+})
+bot.onJoined()
```
Outdated.

devmike123 replied:
My exams are going on. So I am working on it slowly. Thanks for rectifying. It'll help me a lot.

Leref reviewed (changes requested):
It seems your "Fork" is super outdated as there's some files appearing to be from v4 and not from v5 branch.
Please update them accordingly and changes.

devmike123 committed:
Update home.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Create Readme.md

devmike123 committed:
Add files via upload

devmike123 committed:
Update README.md

devmike123 committed:
Update home.md

devmike123 committed:
Update README.md

devmike123 committed:
Added documentation!

I have added the documentation as this file was empty. For any wrong information given, pls contact me.

devmike123 committed:
Update bot.oninteractioncreate.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update README.md

devmike123 committed:
Update home.md

devmike123 committed:
Update home.md

devmike123 committed:
Update README.md

devmike123 committed:
Added v5.0.1 changelogs

devmike123 committed:
Added v5.0.2 release notes

devmike123 committed:
Added v5.0.3 release notes

devmike123 committed:
Added v5.0.4 release notes

devmike123 committed:
Added all release notes

devmike123 committed:
Update bot.oninteractioncreate.md

devmike123 committed:
Update usdchanneltopic.md

devmike123 committed:
Update usdchanneltype.md

devmike123 committed:
Update usdcharcount.md

devmike123 committed:
Update usdcheckcondition.md

devmike123 committed:
Update usdcheckcontains.md

devmike123 committed:
Update usdclear.md

devmike123 committed:
Update usdclearreaction.md
